zip-0000.rst

```diff
@@ -0,0 +1,580 @@
+represent a net improvement. The proposed implementation, if applicable,
+must be solid and must not complicate the protocol unduly.
+
+The ZIP Owner may update the draft as necessary in the git
+repository. Updates to drafts should also be submitted by the Owner
+as pull requests.
+
+
+Transferring ZIP Ownership
+--------------------------
+
+It occasionally becomes necessary to transfer ownership of ZIPs to a new
+Owner. In general, we'd like to retain the original Owner as a
+co-Owner of the transferred ZIP, but that's really up to the original
+Owner. A good reason to transfer ownership is because the original
+Owner no longer has the time or interest in updating it or following
+through with the ZIP process, or has fallen off the face of the 'net
+(i.e. is unreachable or not responding to email). A bad reason to
+transfer ownership is because you don't agree with the direction of the
+ZIP. We try to build consensus around a ZIP, but if that's not possible,
+you can always submit a competing ZIP.
+
+If you are interested in assuming ownership of a ZIP, send a message
+asking to take over, addressed to both the original Owner and the ZIP
+Editors. If the original Owner doesn't respond to email in a timely
+manner, the ZIP Editors will make a unilateral decision (it's not like
+such decisions can't be reversed :).
+
+
+ZIP Editors
+-----------
+
+The current ZIP Editors are Daira Hopwood, representing the Electric Coin
+Company, and Josh Cincinnati, representing the Zcash Foundation. Both
+can be reached at zips@z.cash . The current design of the ZIP Process
+dictates that there are always at least two ZIP Editors: one from the
+Electric Coin Company and one from the Zcash Foundation. Additional Editors may
+be selected by consensus among the current Editors.
+
+
+ZIP Editor Responsibilities & Workflow
+--------------------------------------
+
+The ZIP Editors subscribe to the Zcash Ecosystem Development list.
+
+For each new ZIP that comes in an Editor confirms the following:
+
+* Read the ZIP to check if it is ready: sound and complete. The ideas
+  must make technical sense, even if they don't seem likely to be
+  accepted.
+* The title should accurately describe the content.
+* The ZIP draft must have been sent to the Zcash Ecosystem Development
+  mailing list or another suitable forum for discussion.
+* Motivation and backward compatibility (when applicable) must be
+  addressed.
+* The licensing terms are acceptable for ZIPs.
+
+If the ZIP isn't ready, the editor will send it back to the Owner for
+revision, with specific instructions.
+
+Once the ZIP is ready for the repository it should be submitted as a
+"pull request" to the `ZIPs git repository <https://github.com/zcash/zips>`__
+where it may get further feedback.
+
+The ZIP Editors will:
+
+* Assign a ZIP number in the pull request.
+* Merge the pull request when it is ready.
+* List the ZIP in `README.rst <README.rst>`__
```

Comment:
Currently, this repository only has a `README.md`.

Reply:
I'm inclined to just remove this step. @acityinohio, does that sound reasonable?

Reply:
Totally works for me, updated with that step removed.